website/src/content/development/testing.mdx

```diff
@@ -298,7 +298,7 @@ The `.env` file should be git ignored. On CI, you can define environment variabl
 In the `cypress/plugins/index.js` file, you need to configure the `task` for Custom Applications and to load the environment variables:
 
 ```js title="cypress/plugins/index.js"
-const customApplications = require('@commercetools-frontend/cypress/task');
+const { customApplicationConfig } = require('@commercetools-frontend/cypress/task');
```

Comment:
@kark do you think we can revert changes of this file now?

Reply:
Hi @kark, I thought keeping this would be fine. But I see your point about the mixed imports. And we want to recommend users to use named exports instead. Then, let's leave the change like this.